apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/style.scss

```diff
@@ -19,3 +27,16 @@
 		display: table;
 	}
 }
+
+.template-block__overlay {
+	display: none;
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+
+	.components-placeholder__instructions {
+		color: white;
```

Comment:
<img width="933" alt="Screenshot 2019-06-17 at 16 22 11" src="https://user-images.githubusercontent.com/1182160/59611744-2e18cd80-911c-11e9-8a8f-240d569c42ac.png">

The instructions text is hard to read with white color atm, should we just keep it black?

Reply:
It will really depend on the background. On my tests, back was hard to read. I'm experimenting with the `mix-blend-mode` selector to fix this.

Reply:
didn't work, mostly because the `PlaceHolder` already has some opacity settings. Changed to black on 987235ca3192f5be6d8e31a66516ee15a955a487.